ladybug_geometry/geometry3d/polyface.py

```diff
@@ -155,7 +155,7 @@ def from_faces(cls, faces, tolerance=0):
         face_indices = []  # collection of face indices
         for f in faces:
             ind = []
-            loops = (f.boundary,) if not f.has_holes else (f.boundary,) + f.holes
+            loops = [f.boundary] if not f.has_holes else [f.boundary] + f.holes
```

Comment:
I don't know how this change ended up in your fork but you are reverting an important bug fix that I committed before you started work on the sphere:
https://github.com/ladybug-tools/ladybug-geometry/commit/b11bfb127f1f74b491256babcdb30e30101d29f3

Please change it back to how it was (with tuples) and then we should be good to go.